src/app/core/components/public-settings/public-settings.component.spec.ts

```diff
@@ -32,14 +32,15 @@ const mockApiService = {
 
 describe('PublicSettingsComponent', () => {
   let shallow: Shallow<PublicSettingsComponent>;
+  let messageShown = false;
```

Comment:
Is this value used in tests at all? Its value doesn't seem to be reset in between test runs if it is used.